GrailDiary/GrailDiary/POIDetailViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  POIDetailViewController.swift
+//  GrailDiary
+//
+//  Created by Jonalynn Masters on 9/12/19.
+//  Copyright © 2019 Jonalynn Masters. All rights reserved.
+//
+
+import UIKit
+
+class POIDetailViewController: UIViewController {
+
+    
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var countryLabel: UILabel!
+    @IBOutlet weak var clueListTextView: UILabel!
```

Comment:
One issue I ran into was these outlets not being wired in the storyboard. Without them being wired, it crashes when trying to transition into this view. We are calling updateViews() and setting the text of the labels, but there is no label to display since it hasn't been wired yet.

Another thing is you've subclassed your clueListTextView as a UILabel instead of a UITextView.